{"name": "test_login_0_failed_by_wrong_datas[data0]", "status": "passed", "parameters": [{"name": "data", "value": "{'user': '18684720553', 'passwd': '', 'check': '密码不能为空'}"}], "start": 1562346499776, "stop": 1562346500052, "uuid": "27251d28-de39-4c75-9197-767047777aca", "historyId": "1c4196b33c40eb4ce9335a5ebf3512ec", "fullName": "TestCases.test_login.TestLogin#test_login_0_failed_by_wrong_datas[data0]", "labels": [{"name": "story", "value": "登录失败"}, {"name": "feature", "value": "登录页面功能"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-F99FSHQ"}, {"name": "thread", "value": "9160-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}